examples/transmission_main_example.py

```diff
@@ -144,16 +144,15 @@ def main(network, equipment, source, destination, req = None):
     for dp_db in power_range:
         req.power = db2lin(pref_ch_db + dp_db)*1e-3
         if power_mode:
-            print(f'\nPropagating with input power = {lin2db(req.power*1e3):.2f}dBm :')
+            print(f'\nPropagating with input power = {ansi_escapes.cyan}{lin2db(req.power*1e3):.2f} dBm{ansi_escapes.reset}:')
         else:
-            print(f'\nPropagating in gain mode: power cannot be set manually')
+            print(f'\nPropagating in {ansi_escapes.cyan}gain mode{ansi_escapes.reset}: power cannot be set manually')
         infos = propagate2(path, req, equipment, show=len(power_range)==1)
         if power_mode:
-            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f}dBm :')
+            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f} dBm:')
         else:
             print(f'\nTransmission results:')
-        #info message in gain mode
-        print(destination)
+        print(f'SNR total (at signal bandwidth): {ansi_escapes.cyan}{mean(destination.snr):.02f} dBm{ansi_escapes.reset}')
```

Comment:
Unit for SNR is dB, not dBm.

Reply:
Doh, thanks, fixed. I also updated to captured images (thankfully, it was a text-based format so I didn't have to redo it).